engine/table/src/main/java/io/deephaven/engine/util/LayoutHintBuilder.java

```diff
@@ -105,9 +108,63 @@ static AutoFilterData fromString(String string) {
         }
     }
 
+    private static class ColumnGroup {
+
+        String name;
+        final List<String> children;
+        Color color;
+
+        ColumnGroup(String name, List<String> children, Color color) {
+            NameValidator.validateColumnName(name);
+            children.forEach(c -> NameValidator.validateColumnName(c));
+
+            this.name = name;
+            this.children = children;
+
+            if (color != null) {
+                this.color = color;
```

Comment:
since we leave it null if the passed in color is null, why not just assign it without an if?